doc/nrf/gs_installing.rst

```diff
@@ -372,11 +372,7 @@ The Zephyr Software Development Kit (SDK) contains toolchains for each of Zephyr
 It also includes additional host tools, such as custom QEMU and OpenOCD builds.
 
 .. note::
-
-   Set the following environment variables to ensure the Zephyr SDK is detected correctly:
-
-   * :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``
-   * :envvar:`ZEPHYR_SDK_INSTALL_DIR` to the path of the Zephyr SDK
+   When updating Zephyr SDK, make sure to :ref:`double-check the Zephyr SDK variables <zephyr:toolchain_zephyr_sdk_update>`.
```

Comment:
```suggestion
   When updating Zephyr SDK, make sure to :ref:`verify the Zephyr SDK variables <zephyr:toolchain_zephyr_sdk_update>`.
```
See double-check in https://projecttools.nordicsemi.no/confluence/pages/viewpage.action?spaceKey=TECHDOC&title=Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-D

Reply:
Implemented, please re-check.